src/components/dictionaryConcepts/components/ActionButtons.jsx

```diff
@@ -23,11 +24,21 @@ const ActionButtons = ({
           Edit
           </Link>
           <AddMapping />
+          {
+          mappings && (
+          <ViewConceptMappings
+            mappings={mappings}
+            displayName={display_name}
+            mappingLimit={mappingLimit}
```

Comment:
@dessHub, I think it is best if we do not mix camel case and snake case. in the above, you used a variable name `display_name` in snake case, then you later used `mappingLimit` in camel case. I think it would be best if we used snake case.

Reply:
I did use camel case because it has been our convention. Variables in snake case are the response from the API so I have no control over them.

Reply:
Cool, thanks for the clarity